packages/docusaurus-theme-classic/src/theme/BlogPostItem/Header/Author/index.tsx

```diff
@@ -22,24 +81,36 @@ export default function BlogPostItemHeaderAuthor({
   author,
   className,
 }: Props): JSX.Element {
-  const {name, title, url, imageURL, email} = author;
+  const {name, title, twitter, github, url, imageURL, email} = author;
   const link = url || (email && `mailto:${email}`) || undefined;
+  const renderSocialMedia = () => (
+    <div className={clsx(styles.authorSocial)}>
+      {(twitter as string) && <Twitter user={twitter as string} />}
```

Comment:
uknown keys such as `twitter` and `github` (and other keys that are not predefined) are typed as unknown which triggers TS to yell `Type 'unknown' is not assignable to type 'ReactNode'`
However `as string` might be an ugly solution, otherwise maybe update the `Props` type in `theme classic.d.ts` but which key should we define here ? (twitter?github? other random website?)

Reply:
We shouldn't use unknown keys in the first place

The type should contain a new "socials" attribute and we should validate those socials for both authors.yml and inline authors front matter

We should cover the most common socials (github, twitter/x, LinkedIn, SO, IG, website...) and allow unknown props (`.unknown()` in schema, `[key: string]: string` in type) in case users want to add extra social platforms

Reply:
I've changed the schema to allow a social object, however i'm not sure about the typing of unknown props if the user add a social website